### ISSUE TITLE
[Serious] Elements must meet minimum color contrast ratio thresholds

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#117]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87281 |
| **Checkpoint** | 1.4 Distinguishable › Color Contrast (regular text) (1.4.3.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | Footer |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/dashboard> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Automated |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 04/23/26 at 6:12 pm |
| **Unit Type** | Component |
| **Issue Type** | Accessibility |
| **Reported By** | sharyn.morrow@ablr360.com |

## Description

Elements must meet minimum color contrast ratio thresholds
Following the word Commit, the link text "5b6d7e1" in blue on a light gray background does not meet minimum color contrast requirements.

## Recommendation

Fix any of the following:
  Element has insufficient color contrast of 3.27 (foreground color: #3e8ed0, background color: #f7f7f7, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

## Source Code

```
<a href="https://github.com/APTrust/registry/commit/1cefc5dfa6133c6f1c1862cc678b3bd368a40d62" target="_blank">1cefc5d</a>
```

## Screenshots

![footer blue offwhite contrast](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87281/footer_blue_offwhite_contrast.png)
